datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -1966,7 +1966,7 @@ select
 from t_source;
 
 # No groupy
-query error DataFusion error: Internal error: Min/Max accumulator not implemented for type Binary\. This was likely caused by a bug in DataFusion's code and we would welcome that you file an bug report in our issue tracker
+statement ok
```

Comment:
I think this line should be changed to `query` rather than `statement` -- otherwise this test is simply ensuring the query doesn't error but not checking the results